### DISCUSSION TITLE
multiple animations

### DISCUSSION THREAD

**Discussion by Dovatech** [google/model-viewer#2457] [Q&A]
Hello, I have a question about animation in the model viewer. I want to select and play multiple animations at the same time. Is it possible to do this in model viewer?
### Browser Affected

- [x] Chrome, version: xx.x.xxxx.xx
- [x] Edge
- [ ] Firefox
- [ ] IE
- [ ] Safari

### OS

- [ ] Android
- [ ] iOS
- [ ] Linux
- [ ] MacOS
- [x] Windows

### AR

- [x] WebXR
- [ ] SceneViewer
- [ ] QuickLook

**Comment by elalish** [comment]
No, that's not currently possible. You'll need to export an animation that has the combination you want.

**Reply by Dovatech** [reply]
Ok, if I exoprt model with 2 animations , can I play this animations at the same time in model viewer?

**Reply by elalish** [reply]
Our API only lets you select one animation at a time. You'd need to bake one animation with the moves from both. Or, you can hack your way down into our three.js internals and do whatever you like, but that's more likely to get broken during updates.

**Reply by Dovatech** [reply]
Ok. Thank you
